### PR TITLE
External resources and app name update

### PR DESCRIPTION
**Pull request by michaelbabyn**
This PR address #41 and finishes off #168 by ensuring that `__name__` and `external_stylesheets = ['https://codepen.io/chriddyp/pen/bWLwgP.css']` are initialized in `dash.Dash` for all the examples.

#193 is also addressed here since cases using `append_css` and `append_scripts` have been replaced by `external_stylesheets` and `external_scripts`.

### CODE REVIEW

File: tutorial/examples/excel.py [hunk 1]
```diff
@@ -3,7 +3,13 @@
...
 import dash_html_components as html
 
-app = dash.Dash('excel example')
+external_stylesheets = [
+    'https://codepen.io/chriddyp/pen/bWLwgP.css',
+    'https://codepen.io/chriddyp/pen/LLYbXR.css'
+]
+
+
+app = dash.Dash('excel example', external_stylesheets=external_stylesheets)
```
Comment (T4rk1n) [inline tutorial/examples/excel.py]: `__name__`

File: tutorial/auth.py [hunk 2]
```diff
@@ -272,7 +275,9 @@ def update_graph(dropdown_value):
 APP_URL = 'http://127.0.0.1:8050/'
 
 
-app = dash.Dash('auth')
+external_stylesheets = ['https://codepen.io/chriddyp/pen/bWLwgP.css']
+
+app = dash.Dash('auth', external_stylesheets=external_stylesheets)
```
Comment (T4rk1n) [inline tutorial/auth.py]: `__name__`

File: tutorial/auth.py [hunk 3]
```diff
@@ -161,7 +163,9 @@ def update_graph(dropdown_value):
 APP_NAME = 'Dash Authentication Sample App'
 APP_URL = 'https://my-dash-app.herokuapps.com'
 
-app = dash.Dash('auth')
+external_stylesheets = ['https://codepen.io/chriddyp/pen/bWLwgP.css']
+
+app = dash.Dash('auth', external_stylesheets=external_stylesheets)
```
Comment (T4rk1n) [inline tutorial/auth.py]: `__name__`

**Comment by chriddyp** [comment]
Could we add a section right above "Example: Including Local CSS and JavaScript" that's bolded and says:

> **Important: For these examples, you need to include `__name__` in your Dash constructor**. That is, `app = dash.Dash(__name__)` instead of `app = dash.Dash()`. [Here's why](https://community.plot.ly/t/dash-app-does-not-load-assets-and-app-index-string/12178/10?u=chriddyp).

**Comment by chriddyp** [comment]
otherwise 💃

**Comment by chriddyp** [comment]
@michaelbabyn - I just invited you to our organization's percy account, so you should be able to go into percy and approve the screenshot changes.

**Pull request merged**